Libplanet/Store/LiteDBStore.cs

```diff
@@ -376,6 +376,12 @@ public override IEnumerable<HashDigest<SHA256>> IterateBlockHashes()
         /// <inheritdoc/>
         public override void PutBlock<T>(Block<T> block)
         {
+            // checks whether a block already exists or not.
+            if ((_db.FileStorage.FindById(BlockFileId(block.Hash)) is null) == false)
```

Comment:
How about this?
```suggestion
            if (!(_db.FileStorage.FindById(BlockFileId(block.Hash)) is null))
```

Reply:
I prefer simple and readable codes, but if you want it that way, that's alright.